src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts

```diff
@@ -154,13 +159,34 @@ export var processViewDirective = (
     };
 };
 
-export var listItemDirective = () => {
+export var listItemDirective = (
+    adhConfig : AdhConfig.IService,
+    adhHttp : AdhHttp.Service,
+    adhProcess : Service
+) => {
     return {
         restrict: "E",
+        templateUrl: adhConfig.pkg_path + pkgLocation + "/ListItem.html",
         scope: {
             path: "@"
         },
-        template: "<a data-ng-href=\"{{ path | adhResourceUrl }}\">{{path}}</a>"
+        link: (scope) => {
+            adhHttp.get(scope.path).then((process) => {
+                if (process.data[SIImageReference.nick] && process.data[SIImageReference.nick].picture) {
+                    scope.picture = process.data[SIImageReference.nick].picture;
+                }
+                scope.title = process.data[SITitle.nick].title;
+                scope.processName = adhProcess.getName(process.content_type);
+                if (process.data[SILocationReference.nick] && process.data[SILocationReference.nick].location) {
+                    adhHttp.get(process.data[SILocationReference.nick].location).then((loc) => {
+                        scope.locationText = loc.data[SITitle.nick].title;
+                    });
+                }
+                var workflow = process.data[SIWorkflow.nick];
+                scope.participationStartDate = getStateData(workflow, "participate").start_date;
+                scope.participationEndDate = getStateData(workflow, "evaluate").start_date;
```

Comment:
Update: in theory, all relevant processes have an `evaluate` phase. So, in my opinion, this code is perfect for now, but we might need to modify it later.